lib/std/array_list.zig

```diff
@@ -52,6 +64,7 @@ pub fn ArrayListAligned(comptime T: type, comptime alignment: ?u29) type {
         }
 
         /// Deinitialize with `deinit` or use `toOwnedSlice`.
+        /// Does not cause illegal behavior.
```

Comment:
This goes without saying.

Reply:
Does this apply to all instances of this or only this particular one?

Reply:
All instances